src/features/rewards/disabledPanel/style.ts

```diff
@@ -0,0 +1,55 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this file,
+* You can obtain one at http://mozilla.org/MPL/2.0/.
+*/
+
+import styled from 'styled-components'
+
+export const StyledWrapper = styled<{}, 'div'>('div')`
+  width: 330px;
+  padding: 30px 25px 25px;
+  border-radius: 8px;
```

Comment:
we need to remove it as we can't have radius in panel

Reply:
done